apps/src/codebridge/FileBrowser/prompts/openImportFromBackpackPrompt.tsx

```diff
@@ -36,34 +38,49 @@ export const openImportFromBackpackPrompt = async ({
   projectFiles,
   validationFile,
 }: OpenImportFromBackpackPromptArgsType) => {
-  const handleError = (message: string) => () => {
+  const handleError = (title: string, message: string) => () => {
     // TODO: send analytics / add logging.
-    console.error(message);
+    const bodyComponent = <BackpackErrorAlertBody message={message} />;
+    dialogControl?.showDialog({
+      type: DialogType.GenericAlert,
+      title,
+      bodyComponent,
+    });
   };
   // Delete a file from the backpack.
-  const handleDelete = async (filename: string) => {
+  const handleDelete = async (selectedFileName: string) => {
     backpackApi.deleteFiles(
-      [filename],
-      handleError(`Error in deleting file ${filename}`),
-      () => console.log(`Deleted file ${filename}`)
+      [selectedFileName],
+      handleError(
+        codebridgeI18n.deleteFromBackpackTitle(),
+        codebridgeI18n.deleteFromBackpackError({selectedFileName}) +
```

Comment:
I was getting lint errors when using template strings so using concatenation when there is a key/value https://github.com/code-dot-org/code-dot-org/pull/64118/commits/10f237cf1511da6fbe3d978bf86817bb30532d7a